src/mongocxx/test/spec/unified_tests/operations.cpp

```diff
@@ -899,6 +1042,212 @@ client_session* get_session(document::view op, entity::map& map) {
     return &map.get_client_session(session_name);
 }
 
+document::value run_command(database& db, document::view operation) {
+    document::view arguments = operation["arguments"].get_document().value;
+    document::view command = arguments["command"].get_document().value;
+
+    return db.run_command(command);
```

Comment:
`assert_result` assumes that operations return results in a document with the key `result`. Arguably, results should be in its own class. But for the scope of this PR, I think it should follow that pattern to avoid confusing future test failures, and return a document `{"result": <what run_command returns> }`.

Reply:
Ah gotcha, returned a `{"result": ...}` document instead!